.changeset/small-snakes-guess.md

```diff
@@ -0,0 +1,7 @@
+---
+"@latticexyz/common": patch
+"@latticexyz/world": patch
+"@latticexyz/cli": patch
+---
+
+Added support for namespaced systems.
```

Comment:
can you describe how this should be used?

Reply:
yes this is a WIP, just thinking about how best to put it!